contrib/postprocessing/lethe_pyvista_tools/__init__.py

```diff
@@ -0,0 +1,302 @@
+#############################################################################
+"""Class of methods to post-process lethe results with pyvista"""
+#############################################################################
+
+# Import modules
+import shutil
+import pyvista as pv
+from tqdm import tqdm
+
```

Comment:
Not necessarily. I am doing this because the only class is this one and it has the same name as the folder. But If we had more object types I would maybe create a separate object file or multiple folders.